Makefile.org

```diff
@@ -424,6 +424,14 @@ clean:	libclean
 	rm -fr $$i/*; \
 	done
 
+distclean: clean
+	-$(RM) `find . -type l -a \! -path "./.git/*"`
+	$(RM) apps/CA.pl
+	$(RM) test/evptests.txt test/newkey.pem test/testkey.pem test/testreq.pem
+	$(RM) tools/c_rehash
+	$(RM) crypto/opensslconf.h
+	$(RM) Makefile Makefile.bak
```

Comment:
There is certain asymmetry in the way one can answer such questions. To dismiss something as non-universally available it's sufficient to find *one* system among supported ones that doesn't support a flag. But to confirm something as universally available it takes to test on *all* systems :-( But, as suggested alternative uses only flags from old Single Unix Specification it should be more portable in general. BTW, it seems that I was wrong calling -path as GNU-ism, as it is specified in latest SUS. But as already said/implied, we still support older systems...